.drone.yml

```diff
@@ -196,8 +196,8 @@ pipeline:
       - DOCKER_HOST=tcp://docker:2375
     commands:
       - echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
-      - docker tag owncloud/server:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} owncloud/server:10.2.1-rc3
-      - docker push owncloud/server:10.2.1-rc3
+      - docker tag owncloud/server:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} owncloud/server:10.3.0-alpha
```

Comment:
Even if the tarball is not named like that, so far I have always added alpha1 for all previous testing builds.

Reply:
Now you are pushing alpha1, but you aren't tagging it.

Reply:
Both done